<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>template_form</class>
 <widget class="QDialog" name="template_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
     <property name="title">
      <string>Kaavakohteen tiedot</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="feature_name_label">
        <property name="text">
         <string>Nimi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="feature_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="feature_description_label">
        <property name="text">
         <string>Kuvaus</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="feature_description">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="feature_underground_label">
        <property name="text">
         <string>Maanalaisuus</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CodeComboBox" name="feature_type_of_underground"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Kaavamääräysryhmät</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="regulation_groups_tree_layout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Kaavamääräysryhmäkirjastot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="plan_regulation_group_libraries_combobox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Valitut kaavamääräysryhmät</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="plan_regulation_group_scrollarea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="plan_regulation_group_scrollarea_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>656</width>
             <height>600</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CodeComboBox</class>
   <extends>QComboBox</extends>
   <header>arho_feature_template.gui.components.code_combobox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>template_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
